server/src/routes/collector.routes.js

```diff
@@ -10,4 +11,5 @@ export const routes = (app) => {
   app.get('/types', getTypes);
   app.post('/accounts', getAccounts);
   app.post('/add-tasks', addTasks);
+  app.post('/deleteAccountData', deleteAccountData);

```

Comment:
I would have put `'/delete-account-data'` to be consistent with the rest, what do you think?

Reply:
I don't have the full picture of this issue, could you explain me the need of deleting account data?

Reply:
@sandoche Cool, thanks for reviewing. I pushed a couple of commits based on your feedback. See below.

Reply:
Awesome thanks!